README.md

```diff
@@ -1 +1,5 @@
-# Review-repo
\ No newline at end of file
+# FCC Test Suite Template
+
+**Fork** this project (do not open a Pull Request) to start working on any of freeCodeCamp's responsive web design course projects. Select the project from the dropdown menu and run the tests to see which requirements your project is still missing.
+
+You don't know how to fork a project? [Click hear](https://help.github.com/articles/fork-a-repo/) to learn how to do it.
```

Comment:
Please update the readme according to your project. This is important because when a user lands in your GitHub repository, this is where the user would go first to understand the project. When the readme file is empty or not meaningful, this gives out a bad impression or a poorly maintained repository.